emails/2023-02-10.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+This week, we had [an iconic video](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s) feature us as one of the 15 future databases. Thanks [Fireship](https://www.youtube.com/@Fireship)! For this week's joke, I'm just going to ask [you watch the first 5 seconds of the video](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s). Fireship is funnier and more daring than I could ever be in this email.
+
+### Dolt for Backups
+
+Astute reader will notice a lot more content focused on how and why to use Dolt. We've done a good job building a great product people are interested in.But we haven't done a good enough job explaining how and why to use it. We're making a concerted effort to change that this year, starting with written blogs, because that's what we're best at.
```

Comment:
```suggestion
Astute readers will notice a lot more content focused on how and why to use Dolt. We've done a good job building a great product people are interested in. But we haven't done a good enough job explaining how and why to use it. We're making a concerted effort to change that this year, starting with written blogs, because that's what we're best at.
```